internal/graph/check.go

```diff
@@ -577,7 +578,7 @@ func (c *LocalChecker) ResolveCheck(
 
 	typesys, ok := typesystem.TypesystemFromContext(ctx)
 	if !ok {
-		panic("typesystem missing in context")
+		return nil, fmt.Errorf("%w: typesystem missing in context", openfgaErrors.ErrUnknown)
```

Comment:
Add the test for this one, I believe we talked about it in a different PR.

Reply:
Added tests for all APIs :)